internal/controller/config/rbac/role.yaml

```diff
@@ -15,6 +15,7 @@ rules:
   - list
   - update
   - watch
+  - create
```

Comment:
Jjust making sure, Is that auto generated?

Reply:
No, fixed now.